arangod/IResearch/IResearchKludge.cpp

```diff
@@ -64,46 +64,56 @@ void syncIndexOnCreate(Index& index) {
 
 namespace arangodb::iresearch::kludge {
 
-const char TYPE_DELIMITER = '\0';
-const char ANALYZER_DELIMITER = '\1';
+static constexpr char kTypeDelimiter = '\0';
+static constexpr char kAnalyzerDelimiter = '\1';
+#ifdef USE_ENTERPRISE
+static constexpr char kNestedDelimiter = '\2';
+#endif
 
-std::string_view constexpr NULL_SUFFIX("\0_n", 3);
-std::string_view constexpr BOOL_SUFFIX("\0_b", 3);
-std::string_view constexpr NUMERIC_SUFFIX("\0_d", 3);
-std::string_view constexpr STRING_SUFFIX("\0_s", 3);
+static std::string_view constexpr kNullSuffix{"\0_n", 3};
+static std::string_view constexpr kBoolSuffix{"\0_b", 3};
+static std::string_view constexpr kNumericSuffix{"\0_d", 3};
+static std::string_view constexpr kStirngSuffix{"\0_s", 3};
```

Comment:
move constexpr to static : static constexpr, also you can use anon namespace instead of all static here
for more code style consistency

Reply:
Done